### PR TITLE
how do i pr to your branch

### PR DESCRIPTION
**Pull request by Pepsilawn**
he lp :-(

the code works also i think
image

### CODE REVIEW

File: fulp_modules/features/ert/code/modsuits.dm [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	icon = 'fulp_modules/features/ert/icons/modsuit.dmi'
+	worn_icon = 'fulp_modules/features/ert/icons/modsuit_worn.dmi'
+	// see fulpmodules/Z_edits/modsuits.dm
+	icon_path = 'fulp_modules/features/ert/icons/modsuit.dmi'
+	worn_icon_path = 'fulp_modules/features/ert/icons/modsuit_worn.dmi'
+	cell = /obj/item/stock_parts/cell/hyper
+
+/obj/item/mod/control/pre_equipped/fulp/ert/commander/medical
+	theme = /datum/mod_theme/responsory/fulp/medical
+	initial_modules = list(/obj/item/mod/module/storage/large_capacity, /obj/item/mod/module/flashlight, /obj/item/mod/module/health_analyzer, /obj/item/mod/module/injector)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/ert/code/modsuits.dm]:
```suggestion
	initial_modules = list(
		/obj/item/mod/module/storage/large_capacity,
		/obj/item/mod/module/flashlight,
		/obj/item/mod/module/health_analyzer,
		/obj/item/mod/module/injector,
	)
```

File: fulp_modules/features/ert/code/modsuits.dm [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+/obj/item/mod/control/pre_equipped/fulp/ert/commander/engineering
+	theme = /datum/mod_theme/responsory/fulp/engineering
+	initial_modules = list(/obj/item/mod/module/storage/large_capacity, /obj/item/mod/module/welding, /obj/item/mod/module/rad_protection, /obj/item/mod/module/jetpack, /obj/item/mod/module/flashlight)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/ert/code/modsuits.dm]:
```suggestion
	initial_modules = list(
		/obj/item/mod/module/storage/large_capacity,
		/obj/item/mod/module/welding,
		/obj/item/mod/module/rad_protection,
		/obj/item/mod/module/jetpack,
		/obj/item/mod/module/flashlight,
	)
```

File: fulp_modules/features/ert/code/modsuits.dm [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+/obj/item/mod/control/pre_equipped/fulp/ert/commander/security
+	theme = /datum/mod_theme/responsory/fulp/security
+	initial_modules = list(/obj/item/mod/module/storage/large_capacity, /obj/item/mod/module/welding, /obj/item/mod/module/flashlight, /obj/item/mod/module/jetpack, /obj/item/mod/module/holster)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/ert/code/modsuits.dm]:
```suggestion
	initial_modules = list(
		/obj/item/mod/module/storage/large_capacity,
		/obj/item/mod/module/welding,
		/obj/item/mod/module/flashlight,
		/obj/item/mod/module/jetpack,
		/obj/item/mod/module/holster,
	)
```

File: fulp_modules/features/ert/code/modsuits.dm [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+/obj/item/mod/control/pre_equipped/fulp/ert/commander/clown
+	theme = /datum/mod_theme/responsory/fulp/clown
+	initial_modules = list(/obj/item/mod/module/storage, /obj/item/mod/module/welding, /obj/item/mod/module/flashlight, /obj/item/mod/module/bikehorn)
```
Comment (JohnFulpWillard) [inline fulp_modules/features/ert/code/modsuits.dm]:
```suggestion
	initial_modules = list(
		/obj/item/mod/module/storage,
		/obj/item/mod/module/welding,
		/obj/item/mod/module/flashlight,
		/obj/item/mod/module/bikehorn,
	)
```

File: fulp_modules/features/ert/code/ert_outfit.dm [hunk 5]
```diff
@@ -13,9 +13,8 @@
 /datum/outfit/centcom/ert/commander/medical
 	name = "ERT Commander - Medical"
 
-	suit = /obj/item/clothing/suit/space/hardsuit/ert/commandermed
 	suit_store = /obj/item/gun/energy/e_gun
-	back = /obj/item/storage/backpack/ert/medical
+	back = /obj/item/mod/control/pre_equipped/fulp/ert/commander/medical
```
Comment (JohnFulpWillard) [inline fulp_modules/features/ert/code/ert_outfit.dm:17]: is this a bag? and it'll have all items from backpack_contents?

File: fulp_modules/Z_edits/modsuits.dm [hunk 6]
```diff
@@ -0,0 +1,12 @@
+// Allows us to add fulp-exclusive modsuits without having to add the sprites to tg's mod dmi-s.
+
+/obj/item/mod/control/pre_equipped/fulp
+	var/icon_path
+	var/worn_icon_path
+
+/obj/item/mod/control/pre_equipped/fulp/Initialize(mapload, new_theme, new_skin)
+	. = ..()
+	if (icon_path && worn_icon_path)
```
Comment (JohnFulpWillard) [inline fulp_modules/Z_edits/modsuits.dm:9]: fuck you

**Review by JohnFulpWillard — CHANGES_REQUESTED** [review]
i dont know how modsuits work

File: fulp_modules/features/ert/code/modsuits.dm (hunk 1 above)
Comment (Pepsilawn) [reply fulp_modules/features/ert/code/modsuits.dm]: this is how the tg ones format it but i agree it looks better

File: fulp_modules/features/ert/code/ert_outfit.dm (hunk 5 above)
Comment (Pepsilawn) [reply fulp_modules/features/ert/code/ert_outfit.dm:17]: Yeah thats how it works

File: fulp_modules/Z_edits/modsuits.dm (hunk 6 above)
Comment (Pepsilawn) [reply fulp_modules/Z_edits/modsuits.dm:9]: do you have a better suggestion :-(

**Comment by JohnFulpWillard** [comment]
lets fr*aking go

**Pull request merged**